.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,15 @@
+{
+  "image": "mcr.microsoft.com/devcontainers/universal:2",
```

Comment:
changed that, much faster now

Reply:
Are you not getting this error when creating a new codespace? Because I am.
![image](https://github.com/iTwin/iTwinUI/assets/9084735/98ec3dec-493d-4f70-a8c1-26c1d2e9f802)

I wonder if we need to specify all the things:

```
"hostRequirements": {
  "cpus": 4,
  "memory": "16gb",
  "storage": "16gb"
},
```

Reply:
I'm not getting that error. I can try that change you suggested tho

Reply:
Still getting the same error. I guess lets remove this. We can always manually increase the specs on-demand using "New with options"
![image](https://github.com/iTwin/iTwinUI/assets/9084735/7765f375-f439-4637-9f01-bef05158c583)

Reply:
alright yeah i'll remove them